cleanlab/object_detection/rank.py

```diff
@@ -369,6 +378,23 @@ def _get_iou(bb1: Dict[str, Any], bb2: Dict[str, Any]) -> float:
     return iou
 
 
+def _has_overlap(label_boxes, labels):
+    """This function determines whether each labeled box overlaps with another box of a different class (i.e. virtually the same box having multiple conflicting annotations). It returns a boolean array."""
```

Comment:
`label_boxes` is called `lab_bboxes` in other functions. Regardless, I think function params should be generalized to (`bbox_list`, `labels`) as the function can calculate overlap matrix for any set of boxes/labels.

Reply:
Fixed here: https://github.com/cleanlab/cleanlab/blob/1b4059b2a99b3e2ebed1c24a1ba3a2e5cfa50142/cleanlab/object_detection/rank.py#L380